kilink/views_v2.py

```diff
@@ -19,6 +19,14 @@
 linkode_v2 = Blueprint("linkode_v2", __name__)
 
 
+def external_url_for(*args, **kwargs):
+    """Provide the url for some params, always external.
```

Comment:
```suggestion
    """Provide the full url (including host) for some params.
```
Mini sugerencia porque en el futuro seguro no me voy a acordar que era eso de external :P

Reply:
excelente